app/src/main/java/com/example/admin/lab_56_57/ActivityForLab57Fragment.java

```diff
@@ -0,0 +1,116 @@
+package com.example.admin.lab_56_57;
+
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.Spinner;
+
+import com.example.admin.lab_56_57.data.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ActivityForLab57Fragment extends android.support.v4.app.Fragment {
+
+    private AdapterForLinear adapterForLinear_;
+    private AdapterForRelative adapterForRelative_;
+    private Spinner menuSpinner_;
+    private ArrayAdapter<String> spinnerAdapter_;
+
+    private enum Mode_ {
+        LINEAR("Linear Layout"),
+        RELATIVE("Relative Layout");
+
+        private final String text_;
```

Comment:
For using enum with String it is necessary to create a private member variable and a private constructor must be defined. That is why I created this field